src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -645,6 +659,17 @@ export default {
 			this.showNextSteps = showNextSteps;
 			this.animatedSideSheet = isAnimated;
 		},
+		async initializeNewYearsGoalsIterExp() {
```

Comment:
The experiment and feature flag are already being passed into the props of this component. And the parameters passed to the tracking method here don't match any current experiment on MyKiva. So this code can be removed.